python/arcticdb/version_store/library.py

```diff
@@ -941,6 +959,48 @@ def read_metadata(self, symbol: str, as_of: Optional[AsOf] = None) -> VersionedI
         """
         return self._nvs.read_metadata(symbol, as_of)
 
+    def read_metadata_batch(
+        self, symbols: List[Union[str, ReadInfoRequest]]
+    ) -> List[VersionedItem]:
+        """
+        Reads the metadata of multiple symbols.
+
+        Parameters
+        ----------
+        symbols : List[Union[str, ReadInfoRequest]]
+            List of symbols to read.
+        
+        Returns
+        -------
+        List[VersionedItem]
+            A list of the read results, whose i-th element corresponds to the i-th element of the ``symbols`` parameter.
+            A None object will be returned if either that version doesn't exist, or it exists but there is no metadata
```

Comment:
That isn't the API we put in the wiki page is it? That has `None` iff the version doesn't exist, and a `VersionedItem` with metadata=None if the version does exist

Reply:
oh yeah, it is just the description, had not been updated

Reply:
Updated as:
        List[VersionedItem]
            A list of the read results, whose i-th element corresponds to the i-th element of the ``symbols`` parameter.
            A VersionedItem object with the metadata field set as None will be returned if the requested version of the 
                symbol exists but there is no metadata
            A None object will be returned if the requested version of the symbol does not exist